queries/models/monitoramento/CHANGELOG.md

```
@@ -1,5 +1,11 @@
 # Changelog - monitoramento
 
+## [1.9.1] - 2026-01-29
+
+### Adicionado
+
+- Adiciona `status` de Não licenciado para `tipo_veiculo` rodoviário no modelo `aux_veiculo_dia_consolidada`  [Processo n. X] (https://github.com/prefeitura-rio/pipelines_rj_smtr/pull/1198)
```

markdownlint-cli2 (0.20.0)

[warning] Bare URL used

(MD034, no-bare-urls)